pytorch/pytorch_edgeml/graph/rnn.py

```diff
@@ -1065,14 +1065,33 @@ def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType):
         self.hiddenDim0 = hiddenDim0
         self.hiddenDim1 = hiddenDim1
         self.outputDim = outputDim
-        supportedCells = ['LSTM']
+        supportedCells = ['LSTM','FastGRNNCell','GRULRCell']
         assert cellType in supportedCells, 'Currently supported cells: %r' % supportedCells
         self.cellType = cellType
+        
         if self.cellType == 'LSTM':
             self.rnnClass = nn.LSTM
-
-        self.rnn0 = self.rnnClass(input_size=inputDim, hidden_size=hiddenDim0)
-        self.rnn1 = self.rnnClass(input_size=hiddenDim0, hidden_size=hiddenDim1)
+            self.rnn0 = self.rnnClass(input_size=inputDim, hidden_size=hiddenDim0)
+            self.rnn1 = self.rnnClass(input_size=hiddenDim0, hidden_size=hiddenDim1)
+        
+        if self.cellType == 'FastGRNNCell':
+            self.rnnClass = FastGRNN
+            self.rnn0 = self.rnnClass(input_size=inputDim, hidden_size=hiddenDim0, gate_non_linearity = fastCellArgs['gate_non_linearity'],
```

Comment:
Please use PEP8 to break lines properly and format properly.

Reply:
More importantly, this is bad code. What if the user did not provided `gate_non_linearity` and instead want to use the default arguments in FastCells. This step will fail with a `KeyNotFound` error.